src/components/modals/FormStepSaveModal.stories.js

```diff
@@ -20,7 +21,7 @@ const mockDestroySessionDELETE = rest.delete(
 export default {
   title: 'Private API / FormStepSaveModal',
   component: FormStepSaveModalComponent,
-  decorators: [ConfigDecorator], // DeprecatedRouterDecorator],
+  decorators: [FormikDecorator, ConfigDecorator], // DeprecatedRouterDecorator],
```

Comment:
I don't think we need `FormikDecorator`, because the component itself already uses `<Formik>` inside, no?